app/konnect/runtime-manager/consumer-groups.md

```diff
@@ -0,0 +1,86 @@
+---
+title: Create Consumer Groups in Konnect
+content_type: tutorial
+badge: enterprise
+---
+With consumer groups, you can define any number of rate limiting tiers and
+apply them to subsets of consumers, instead of managing each consumer
+individually.
+
+For example, you could define three consumer groups:
+* A "gold tier" with 1000 requests per minute
+* A "silver tier" with 10 requests per second
+* A "bronze tier" with 6 requests per 30 seconds
+
+The `consumer_groups` endpoint works together with the [Rate Limiting Advanced plugin](/hub/kong-inc/rate-limiting-advanced).
+
+Consumers that are not in a consumer group default to the Rate Limiting advanced
+plugin’s configuration, so you can define tier groups for some users and
+have a default behavior for consumers without groups.
+
+To use consumer groups for rate limiting, you must:
+* Create one or more consumer groups
+* Create consumers
+* Assign consumers to groups
+* Configure the Rate Limiting Advanced plugin globally with the `enforce_consumer_groups`
+and `consumer_groups` parameters, setting up the list of consumer groups that
+the plugin accepts
+* Configure a rate limiting policy for each consumer group, overriding the 
+plugin's global configuration
+
+## Set up a consumer group with consumers
+
+In this section, you will create the Bronze tier consumer group with a consumer assigned to the group.
+
+1. In {{site.base_short_name}}, click **Runtime Manager** in the sidebar.
+1. Click the **default** runtime group.
+1. From the menu, click **Consumers**, then click **Add consumer**.
+1. Enter a **Username** and **Custom ID**. For this example, you can use "Amal" for each field.
+1. Click **Save**.
+1. Click the **Consumer Groups** tab.
```

Comment:
I think you should split these two instruction sets. The first one created a consumer, and should belong in an H2 about creating a consumer, and the next one adds it to a group.